src/config/cache_fallback.php

```diff
@@ -1,12 +1,66 @@
 <?php
 
 return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Cache Driver Fallback Order
+    |--------------------------------------------------------------------------
+    |
+    | Here you may list all the possible cache drivers you want to use,
+    | in the order you want to fall back to them on. The package will go
+    | down the list until it finds a working cache driver.
+    |
+    */
+
     'fallback_order' => [
         'redis',
         'memcached',
         'database',
         'cookie',
         'file',
         'array'
-    ]
+    ],
```

Comment:
In terms of the order is this something we can somehow configure via envs? How would you envision doing that?

Reply:
Yeah, we'll be publishing the config in our own app, and then from there we can override settings - this is just the default if no envs are used.